### ISSUE TITLE
System reboots 85% complete during Catalina Install

### ISSUE THREAD

**Issue by carloperatoner** [chris1111/HP-Probook-EliteBook-Package-Creator#39]
Thank you for the excellent setup.     I have got further with your setup than with any other method.   I am very close to getting the system to install.   I have one major issue.

I have a 9470m 16G ram  250GB samsung ssd.  Bios F.73

I have followed all of the directions on the website and the video.   I get to the installed and it is about 85% complete.   The system reboots.    When clover boots I get two instances, the installer USB and the new catalina version on the 250GB samsung drive.   The new catalina version does not boot.

Where can i find the install logs to troubleshoot why the system is rebooting?  I have disabled the intel driver as in the last post with the same result.

Thank you.

**Comment by chris1111** [comment]
85% complete Install of macOS ?

**Comment by carloperatoner** [comment]
Yes as the installation proceeds, it says 2 minutes left.   The status bar is about 85% across the screen.   The machine suddenly reboots and you are back to the clover boot menu.   The internal drive is not bootable and it appears that the complete OS install has failed.   Files are loaded on the internal drive and clover shows two boot options.   Boot macos from Internal and boot os from Install macOS Catalina.

**Comment by chris1111** [comment]
Didi you try to boot on Internalk from USB because the install of macOS will never boot without the Post install Clover package

and if the SSD is not bootable try to reinstall from another USB port then don't forgot to format the entire SSD before install macOS

**Comment by carloperatoner** [comment]
Thank you.   I will give it a try.

**Comment by carloperatoner** [comment]
Chris,

Sorry to not have got back earlier.     I was finally successful during the post install.    I have a functioning system but have a few comments/questions:

System:  HP EliteBook 9470M
                16GB Ram
                250GB Samsung SSD


  1.  Where can I donate money for your work?
  2.  When I did the install I did not enable the SSDT generator.    It seemed to impact the boot process where the boot progress bar was distorted when the system boots.   (It would not load the os.)
  3.  Since I am installing on a SSD  I did use the terminal to set sudo trimforce enable.
  4.  Everything works but WIFI.  I did install the AirportBrcmFixup package.   I do not have the Broadcom card  (it is ordered and will be here in a few days.)  I assume  (probably incorrectly) that when I install it, the system will automatically recognize the card.

Your work is excellent.   It made a very complicated process very easy.    Thank you for your work and your help.

Carlo

**Comment by chris1111** [comment]
Thank you for report 
You can put your EFI folder on ZIP here I will see if all is ok
For donation its here [Click to the Image for Donate ](https://www.hackintosh-montreal.com/h37-paypal)

**Comment by carloperatoner** [comment]
Chris,

HP 9470M
HP4000
16G Ram
Samsung 256 GB SSD

Thanks again.   One issue I noticed was that the headphones work, but not the internal speakers.   I think I have a IDT 92HD87 codec.  I did add to the config.plist the following lines and tried, 3, 12, and 13.
                                <key>Audio</key>
                                <dict>
                                                <key>Inject</key>
                                                <integer>13</integer>
                                </dict>

There was no change to the internal speakers.   I am sure I am missing something simple.

Injection values are based on this file: https://github.com/acidanthera/AppleALC/wiki/Supported-codecs

Here is the link to my EFI file.    Let me know if you see anything that will give me trouble.
https://1drv.ms/u/s!Andi_Cu2pecdg_9RI4xn2G-lPV62mg?e=QLoPOt

Thank you again,

Carlo

**Comment by chris1111** [comment]
Thank you very much  for  the donation!
I not see anything wrong only one thing your not have SSDT.aml ? Why ? Do you have any issue with the SSDT ?

For the Audio if you have trouble with AppleALC you can use VoodooHDA 292 
Option 1  UEFI/ESP /macOS Catalina
https://github.com/chris1111/VoodooHDA-2.9.2-Clover-V15

If you use VoodooHDA make sure AppleALC.kext is remove from your /CLOVER/kexts/10.15

**Comment by carloperatoner** [comment]
Chris,

Thank you for your help.  It is greatly appreciated.

I did not load the SSDT since it seemed to impact the video resolution when booting.    The progress bar was distorted with the SSDT so I just did not select it and the system works great.

I downloaded and ran the VoodooHDA 2.92.   It was easy to install, but I have not internal speaker setting and none of the selections allow audio to come out of the speakers.   The headphone works just fine.   I assume that somewhere I need to modify which card I am using.

The device ID is 76E0
The vendor ID is 111D

Can you lend a hand?

Thank you,

Carlo

**Comment by chris1111** [comment]
See the setup here try to select the right choice
![Screen Shot 2020-05-14 at 4 03 07 PM](https://user-images.githubusercontent.com/6248794/81980450-9f971f80-95fc-11ea-95c1-4895d413d712.png)

**Comment by carloperatoner** [comment]
Chris,

I only have one choice.

/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/HDEF@1B

Should I see other options?  I did use option 1.


Carlo

**Comment by chris1111** [comment]
This is really strange if you are not trying to modifying the original AppleHDA.kext by some third party script or app its verry rare voodoohda not working on HP ProBook
I am stunt

**Comment by carloperatoner** [comment]
Chris,

I tried again and suddenly I was able to get it to work.   The volume is not that loud, but if I set to the max it is ok.    I think I am fine now.

One final thing.  Can you point me in the direction of how to change the default boot disk and set a 5 second timer?   There are so many different directions on the web and I know from experience that if you change the config file incorrectly the system will not boot easily.

Thank you.

Carlo

**Comment by chris1111** [comment]
Great! 😃
Setup a boot drive its Verry easy
Open the config.plist with textedit.app
Go to Boot  
see the picture
![Capture d’écran, le 2020-05-15 à 00 38 03](https://user-images.githubusercontent.com/6248794/82012026-b2cddd80-9644-11ea-9a4c-eabe1486f753.png)

**Comment by carloperatoner** [comment]
Done  and done thank you.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by chris1111** [comment]
I am glad to have helped you solve the problems the more you are a good person and it helps greatly 
Thank you @carloperatoner

**Comment by carloperatoner** [comment]
Chris,

My wifi card arrived today.     BCM94352HMB<https://www.ebay.com/itm/Alienware-17-Wireless-WIFI-Card-AzureWave-BCM94352HMB/253983560372?ssPageName=STRK%3AMEBIDX%3AIT&_trksid=p2057872.m2749.l2649>  I plugged it in and it recognizes the card, but I cannot turn on the WiFi and Bluetooth is not recognized.    Before I reinstall Catalina (one more time) is there something I should look at to see why the wifi won’t turn on?

I did check the bios and it shows both Bluetooth and Wifi are enabled.  Rebooted several times.   My next thought is that during the install if it did not see the card it did not enable the software.

Thank you,

Carlo

**Comment by chris1111** [comment]
put this kext on 10.15 then reboot

If wifi ok I iiil gives you Bluthoot kext
[10.15.zip](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/files/4637099/10.15.zip)

**Comment by carloperatoner** [comment]
Thanks,

It did not work, so I will reinstall one last time.    I’m getting pretty good at reinstalling.   😊

Thanks again,

Carlo

**Comment by carloperatoner** [comment]
Chris,

Last email here.    I reinstalled everything and the WiFi and Bluetooth all work fine without the need for the patch in this email.  I had to apply the sound patch to get the speakers working.

One thing to note on my installation.   If I installed the SSDT Generator in the Post installation process the boot would fail and go into a panic loop.   This may be a result of a higher version level of BIOS in my laptop.

Certainly this is a very easy installation process once you do it about 20 times.   😊

Thanks again for your help.

Carlo
